index.html

```diff
@@ -0,0 +1,1115 @@
+    <link rel="stylesheet" href="assets/style/main.css" />
+    <script
+      src="https://kit.fontawesome.com/013fb8bd39.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+    <title>TNW Clone</title>
+  </head>
+  <body>
+    <!--Header-->
+    <header>
+      <div class="fixed-min-logo">
+        <div class="header-logo-fixed">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 66 16">
+            <path
+              fill="#FFF"
+              d="M34 16h-6v-5.2c0-.7-.3-1.3-.7-1.8L25 6.7c-.5-.4-1.1-.7-1.8-.7H22v10h-6V0h8.7c1.3 0 2.6.5 3.5 1.5l4.3 4.3c1 .9 1.5 2.2 1.5 3.5V16zM60 0v5.2c0 .7-.3 1.3-.7 1.8L57 9.3c-.5.5-1.1.7-1.8.7h-2.5c.8-.9 1.3-2.1 1.3-3.3V0h-6v5.2c0 .7-.3 1.3-.7 1.8L45 9.3c-.5.5-1.1.7-1.8.7H42V0h-6v16h8.7c1.3 0 2.6-.5 3.5-1.5l2.8-2.8V16h5.7c1.3 0 2.6-.5 3.5-1.5l4.3-4.3c1-.9 1.5-2.2 1.5-3.5V0h-6zM14 0H0v6h4v10h6V6h4"
+            />
+          </svg>
+        </div>
+        <p>TNW Programs</p>
+      </div>
+      <div class="navigation">
+        <nav class="navbar-left">
+          <ul>
+            <li><a href="#" class="navbarlink">News</a></li>
+            <li><a href="#" class="navbarlink">Events</a></li>
+            <li><a href="#" class="navbarlink">Business</a></li>
+            <li><a href="#" class="navbarlink">AMAs</a></li>
+            <li><a href="#" class="navbarlink">Spaces</a></li>
+            <li><a href="#" class="navbarlink">Terms and Conditions</a></li>
+          </ul>
+        </nav>
+        <nav class="navbar-right">
+          <ul>
+            <li><a href="#" class="navbarlink">About</a></li>
+            <li><a href="#" class="navbarlink">Advertise</a></li>
+            <li><a href="#" class="navbarlink">Jobs</a></li>
+            <li><a href="#" class="navbarlink">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <!-- STICKY HEADER PART -->
+    <section class="stickyheader">
+      <nav class="stickyleft">
+        <ul>
+          <li><a href="#" class="stickylink">LATEST</a></li>
+          <li><a href="#" class="stickylink">HARD FORK</a></li>
+          <li><a href="#" class="stickylink">PLUGGED</a></li>
+          <li><a href="#" class="stickylink">README</a></li>
+          <li><a href="#" class="stickylink">GROWTH QUARTERS</a></li>
+          <li><a href="#" class="stickylink">SHIFT</a></li>
+          <li><a href="#" class="stickylink">NEURAL</a></li>
+        </ul>
+      </nav>
+      <button class="stickyright disappear">
+        <i class="fas fa-bars"></i>
+      </button>
+      <button class="stickyright">
+        <i class="fas fa-search"></i>
+      </button>
+    </section>
+    <!-- FIXED ELEMENTS BOTTOM SIDE-->
+    <div class="fingerprint">
+      <i class="fas fa-fingerprint fa-2x"></i>
+    </div>
+    <div class="invalid">
+      <span class="iconify" data-icon="cil:disabled" data-inline="false"></span>
+    </div>
```

Comment:
**Wrap these sections inside the `header` tag.**

Reply:
The sticky element will stick only to the parent element, so here it should be a child of body, not header